src/components/views/createplayer/LandingPage.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react"
+import { usePlayers } from "../../../hooks/api/usePlayers"
```

Comment:
2. Carmen und moris haben sich meega mühe gegeben für den ersten design Entwurf, und er hat etwas verspieltes, was zum jass spiel passt. Und er ist auch schön, wir haben ihn einfach noch nicht umgesetzt. Wollen wir jetzt wirklich nochmals drüber und alles über den haufen werfen?
Könnten wir eventuell einen Kompromiss finden, mit dem wir das bestehende aufmotzen und schritt für schritt vorgehen können? (So sollte man fast alles auf einmal umbauen, weil sonst ist der wechsel gar stark)

Reply:
Das ist definitiv zu diskutieren...es hat sich einiges geändert. Allen voran der Hintergrund...es war einfach mal ein Vorschlag. Ich denke, es lohnt sich uf jeden Fall noch mehr Zeit in das Design und die Usability und die Nutzerführung zu investieren.

Reply:
Ist mir vorallem wichtig die Diskussion zu starten, denn wir haben noch einige Userstories im Stack...